C#/C#.sublime-syntax

```diff
@@ -1330,43 +1330,12 @@ contexts:
             2: punctuation.separator.cs
             3: punctuation.section.brackets.end.cs
             4: keyword.operator.pointer.cs
-    - match: \((?=(?:[^,)(]|\([^\)]*\))*,)
-      scope: punctuation.section.group.begin.cs
-      push:
-        - meta_scope: meta.group.tuple.cs
-        - match: \)
-          scope: punctuation.section.group.end.cs
-          pop: true
-        - match: ','
-          scope: punctuation.separator.tuple.cs
-        - match: ':'
-          scope: keyword.operator.assignment.cs
-        - match: _\b
-          scope: variable.language.deconstruction.discard.cs
-        - match: (?!{{reserved}})(?={{namespaced_name}}{{type_suffix}}\s+{{name}}\s*[:,])
-          push: var_declaration_explicit
-        - match: '({{name}})(<)'
-          captures:
-            1: support.type.cs
-            2: meta.generic.cs punctuation.definition.generic.begin.cs
-          push:
-            - meta_content_scope: meta.generic.cs
-            - match: ','
-              scope: punctuation.separator.type.cs
-            - match: '>'
-              scope: meta.generic.cs punctuation.definition.generic.end.cs
-              pop: true
-            - include: type
-        - include: line_of_code_in_no_semicolon
     - match: \(
       scope: punctuation.section.group.begin.cs
-      push:
-        - meta_scope: meta.group.cs
-        - match: \)
-          scope: punctuation.section.group.end.cs
-          pop: true
-        - include: lambdas
-        - include: line_of_code_in_no_semicolon
+      branch_point: tuple_or_group
```

Comment:
I'd probably vote for using the same name scheme as for contexts and keep underscore use reserved for variables only. It makes it easier to give them the same name as a context, as already noted in another PR.

Reply:
Nevermind. I just see this syntax uses underscors for contexts, too. So this name is very fine.